src/org/zalando/stups/mint/worker/job/run.clj

```diff
@@ -68,11 +67,11 @@
                     (throw e))
                   (storage/update-status storage-url app-id
                                                      {:has_problems true
-                                                      :s3_errors    (when (s3-exception? e)
+                                                      :s3_errors    (when (storage-exception? e)
                                                                       (inc (:s3_errors mint-app)))
                                                       :message      (str e)}
                                                      tokens)
-                  (if (s3-exception? e)
+                  (if (storage-exception? e)
```

Comment:
In general, it's not a good practice to check exception type like this. Please take a look at: https://github.com/scgilardi/slingshot